workalendar/europe/germany.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
 from datetime import date, timedelta
 from workalendar.core import WesternCalendar, ChristianMixin
+from workalendar.registry import iso_register
 
 
+@iso_register('DE')
```

Comment:
Why uppercase?

Reply:
Everywhere that I see ISO codes mentioned they are upper case. I thought it's a standard

Reply:
Ok fair enough :)

Reply:
ISO standard `de_DE`, where `DE` is Germany (country) and `de` is German (language)

country codes always upper case, and language codes always lower case

Reply:
Yes but because we are speaking only of countries here, we don't need the language code.

Reply:
Forgot to mention, the ISO standard (ISO-3166-1) have 3 options, ALPHA-2 (`DE`), ALPHA-3 (`DEU`), and Numerical (`276`), regional codes are ISO-3166-2 and 3166-3 

[List of ISO-3166-1 codes](http://kirste.userpage.fu-berlin.de/diverse/doc/ISO_3166.html)

Reply:
Since it is valid to combine any language with any country code, and i18n is handled differently and have little to nothing with looking up in calendars, lower case language code can safely be ignored.

Though it should be possible to look up ISO codes for each calendar with all three standards (ALPHA-2, ALPHA-3, Numerical)

Reply:
I propose to use two letter codes (ALPHA-2) for countries and specific ISO-3166-2: 1 (e.g. BO-L`), 2 (e.g.`DE-NW`) or 3 letters (e.g. `GB-WOR`). And I suggest to keep in documentation list of supported regions along with their iso codes.
Current implementation only requires to have codes in format `XX` or `XX-YY` (`(\w)+(-\w+)?`)